requirements.yml

```diff
@@ -17,7 +17,7 @@
 
 - name: openmicroscopy.omero-web
   src:  https://github.com/openmicroscopy/ansible-role-omero-web.git
-  version: 1.0.0-m1
```

Comment:
I assume this requirement bump was also used for https://github.com/openmicroscopy/prod-playbooks/pull/3?

Reply:
Yes sir. It was noticed on this system, and it looks like worked-around on ome-demoserver via https://github.com/kennethgillen/prod-playbooks/blob/c6455280cbebb20f84d1c8392ce78fb48c3e6785/ome-demoserver.yml#L161

Reply:
So the upgrade to ome-demoserver was uneventful, but the change to the Nightshade ecosystem caused a broken omero-web. This was the reason.